mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_position.test

```diff
@@ -0,0 +1,20 @@
+--echo
+--echo # MDEV-27037 mysqlbinlog emits a warning when reaching EOF before stop-condition
+--echo
+
+--let assert_file= $MYSQLTEST_VARDIR/tmp/warn_pos_test_file.out
+--let data_file= $MYSQLTEST_VARDIR/std_data/master-bin.000001
+
+--echo Case: Stop position before EOF, WARNING must not appear
+--echo MYSQL_BINLOG --short-form --start-position=4 --stop-position=97 mysql-test/std_data/master-bin.000001 --result-file=warn_pos_test_file.out 2>&1
```

Comment:
Nice and simple test, I like it :) Do we still need the `2>&1` for the warning to show up in the test result output? Let's also clean up the result file at the end of the test (with MTR command `--remove_file`).

Reply:
Thanks! 😊 You're right, we no longer need `2>&1`.  Didn't know about `--remove_file`, will clean it up.

Reply:
Oops, I spoke too soon; we need `2>&1` 😅